github_token.txt.example

```diff
@@ -0,0 +1 @@
+f0b629d28bcecb6ae78cfa056818b3bd70bcad01
```

Comment:
Not the actual token.

Also @sherson, at some point once we merge this we should replace with a token belonging to @umasstransit, not me.

Reply:
> Also @sherson, at some point once we merge this we should replace with a token belonging to @umasstransit, not me.

Already done.